pre-edited-cs/data/ScriptSource/ArmsItem.txt

```diff
@@ -319,26 +317,25 @@ Equipped the Whimsical Star.<NOD<WAI0003<FRE<EVE5038
 <MSG<TURYour tie to Curly Brace, the only
 warrior you would trust your back to.
 Surely you will meet again one day...<WAI9999<END
-
 #7002
 <MLP<END
-
 #7018
 <KEY<MSGDo you want to remove it?
 <YNJ0000<EQ-0001<FL-0742<MSG
 Deequipped Booster v0.8.<NOD<WAI0003<FRE<EVE5018
+#7021
+<KEY<MSGDo you want to remove it?
```

Comment:
could you add a `<TUR` to each of these remove events while you're in here?

Reply:
oh and to the booster equips I think those don't have em either

Reply:
Should I also add a `<TUR` to the Life Pot prompt while I'm at it?

Reply:
yeah anything in the inventory that's missing em tbh